demo/src/main/java/com/midtrans/demo/DemoConfigActivity.java

```diff
@@ -542,6 +541,8 @@ public void onCheckedChanged(CompoundButton compoundButton, boolean checked) {
             public void onCheckedChanged(CompoundButton compoundButton, boolean checked) {
                 if (checked) {
                     cardClickTitle.setText(R.string.credit_card_type_two_clicks);
+                    secureEnabledSelection.setChecked(true);
+
```

Comment:
it will automatically change the title when call the method setOnCheckedChangeListener